cms/static/js/models/group_configuration.js

```diff
@@ -87,23 +88,35 @@ function(Backbone, _, str, gettext, GroupModel, GroupCollection) {
                 };
             }
 
-            if (attrs.groups.length < 1) {
+            if (!this.canBeEmpty && attrs.groups.length < 1) {
                 return {
                     message: gettext('There must be at least one group.'),
                     attributes: { groups: true }
                 };
             } else {
                 // validate all groups
-                var invalidGroups = [];
+                var validGroups = new Backbone.Collection(),
+                    invalidGroups = new Backbone.Collection();
                 attrs.groups.each(function(group) {
-                    if(!group.isValid()) {
-                        invalidGroups.push(group);
+                    if (!group.isValid()) {
+                        invalidGroups.add(group);
+                    } else {
+                        validGroups.add(group);
                     }
                 });
-                if (!_.isEmpty(invalidGroups)) {
+
+                if (!invalidGroups.isEmpty()) {
                     return {
                         message: gettext('All groups must have a name.'),
```

Comment:
@catong - I've modified the code to keep the existing validation error message when the user does not provide a name for a content group or group within a group configuration here.